staging-service-shared/src/main/java/com/github/cafdataprocessing/services/staging/dao/filesystem/BatchNameProvider.java

```diff
@@ -39,7 +41,9 @@ public static String getSubBatchName()
 
     private static String getCurrentTimeAsString()
     {
-        return cleanseTime(Instant.now().toString());
+        final DateTimeFormatter formatToday = DateTimeFormatter.ofPattern(DATE_TIME_ISO_PATTERN).withZone(ZoneOffset.UTC);
```

Comment:
This will construct the formatter before each use.  It should only need to be constructed once.